core/src/main/scala/org/ensime/core/debug/NoActiveVirtualMachineException.scala

```diff
@@ -0,0 +1,7 @@
+package org.ensime.core.debug
+
+/**
+ * Represents the scenario when a virtual machine is requested but there is
+ * no active virtual machine available.
+ */
+class NoActiveVirtualMachineException extends Exception
```

Comment:
I'm not particularly keen on one line source files. Is there somewhere standard you could gather all your domain objects?